### ISSUE TITLE
segfault when setting jac_penality too low

### ISSUE THREAD

**Issue by jgillis** [casadi/casadi#1751]
``` python
from casadi import *

n = 10

x = MX.sym("x",n,n)
y = MX.sym("y",n,n)
z = MX.sym("z")

f = Function("f",[x,y,z],[mtimes(x,y),x*y+z,z],{"jac_penalty":0.1})
f.fullJacobian()
```

**Comment by jgillis** [comment]
(reproduces in 2.4.*)

**Comment by jgillis** [comment]
Seems to be fixed in 3.5
